Gemfile.lock

```diff
@@ -71,73 +71,73 @@ GEM
       jsonapi-renderer (>= 0.1.1.beta1, < 0.3)
     active_storage_base64 (2.0.0)
       rails (>= 7.0)
-    activejob (7.0.2.3)
-      activesupport (= 7.0.2.3)
+    activejob (7.0.4)
+      activesupport (= 7.0.4)
       globalid (>= 0.3.6)
-    activemodel (7.0.2.3)
-      activesupport (= 7.0.2.3)
-    activerecord (7.0.2.3)
-      activemodel (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
-    activestorage (7.0.2.3)
-      actionpack (= 7.0.2.3)
-      activejob (= 7.0.2.3)
-      activerecord (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
+    activemodel (7.0.4)
+      activesupport (= 7.0.4)
+    activerecord (7.0.4)
+      activemodel (= 7.0.4)
+      activesupport (= 7.0.4)
+    activestorage (7.0.4)
+      actionpack (= 7.0.4)
+      activejob (= 7.0.4)
+      activerecord (= 7.0.4)
+      activesupport (= 7.0.4)
       marcel (~> 1.0)
       mini_mime (>= 1.1.0)
-    activesupport (7.0.2.3)
+    activesupport (7.0.4)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 1.6, < 2)
       minitest (>= 5.1)
       tzinfo (~> 2.0)
-    acts_as_tenant (0.5.1)
+    acts_as_tenant (0.5.2)
       rails (>= 5.2)
       request_store (>= 1.0.5)
-    addressable (2.8.0)
-      public_suffix (>= 2.0.2, < 5.0)
-    airbrussh (1.4.0)
+    addressable (2.8.1)
+      public_suffix (>= 2.0.2, < 6.0)
+    airbrussh (1.4.1)
       sshkit (>= 1.6.1, != 1.7.0)
     annotate (3.2.0)
       activerecord (>= 3.2, < 8.0)
       rake (>= 10.4, < 14.0)
-    apipie-rails (0.7.0)
+    apipie-rails (0.8.2)
       actionpack (>= 5.0)
       activesupport (>= 5.0)
     ast (2.4.2)
     aws-eventstream (1.2.0)
-    aws-partitions (1.573.0)
-    aws-sdk-core (3.130.0)
+    aws-partitions (1.632.0)
+    aws-sdk-core (3.150.0)
       aws-eventstream (~> 1, >= 1.0.2)
       aws-partitions (~> 1, >= 1.525.0)
       aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
-    aws-sdk-kms (1.55.0)
+      jmespath (~> 1, >= 1.6.1)
+    aws-sdk-kms (1.58.0)
       aws-sdk-core (~> 3, >= 3.127.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.113.0)
+    aws-sdk-s3 (1.114.0)
       aws-sdk-core (~> 3, >= 3.127.0)
       aws-sdk-kms (~> 1)
       aws-sigv4 (~> 1.4)
-    aws-sigv4 (1.4.0)
+    aws-sigv4 (1.5.1)
       aws-eventstream (~> 1, >= 1.0.2)
-    bcrypt (3.1.17)
+    bcrypt (3.1.18)
     better_errors (2.9.1)
       coderay (>= 1.0.0)
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
     bindex (0.8.1)
     binding_of_caller (1.0.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.11.1)
+    bootsnap (1.13.0)
```

Comment:
### [bootsnap](https://github.com/Shopify/bootsnap)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.11.1 → 1.13.0

[_[change-log](https://github.com/Shopify/bootsnap/blob/main/CHANGELOG.md), [source-code](https://github.com/Shopify/bootsnap), [gem-diff](https://my.diffend.io/gems/bootsnap/1.11.1/1.13.0)_]





<details>
<summary>Commits</summary>

A change of **21** commits. See the full changes on [the compare page](https://github.com/Shopify/bootsnap/compare/v1.11.1...v1.13.0).

These are the first 10 commits:
- (3044e5f) [Fix invalid path value](https://github.com/Shopify/bootsnap/commit/3044e5f8be6bd1e4724531a0b35262d2d822a35e)
- (163b718) [Reduce the cache key to RUBY_REVISION + RUBY_PLATFORM](https://github.com/Shopify/bootsnap/commit/163b718c078c5f6d5c691f482be46ef21cecf24e)
- (57d883d) [Merge pull request #412 from Shopify/refactor-cache-key](https://github.com/Shopify/bootsnap/commit/57d883d7b04489fcbfba6e93cef405ead29194fa)
- (135eb5c) [Also precompile Rake files](https://github.com/Shopify/bootsnap/commit/135eb5c90f633168c0413bcb54a20942fc17094f)
- (4da3a4f) [test_helper: support platforms where GC compaction is not su…](https://github.com/Shopify/bootsnap/commit/4da3a4f5eb6eab27af68c56a959502437f35c94b)
- (8d0a569) [Merge pull request #414 from terceiro/gc-compact-unsupported…](https://github.com/Shopify/bootsnap/commit/8d0a56930f27c239226a77959dc95b3bb16e4c2a)
- (3e75bf9) [Get rid of the Module#autoload alias method chain](https://github.com/Shopify/bootsnap/commit/3e75bf97dae8c10dd0071efaa18ed65b76984fe6)
- (aeb33aa) [Merge pull request #417 from Shopify/no-autoload-ext](https://github.com/Shopify/bootsnap/commit/aeb33aac0e10cecdfa595f7bf7c9b4b3f2ed379f)
- (2709c4e) [Update CHANGELOG.md](https://github.com/Shopify/bootsnap/commit/2709c4eaf279411192507df3de04bd8ec88fd67a)
- (f213904) [Release 1.12.0](https://github.com/Shopify/bootsnap/commit/f2139046daecb59bdbcc68f0398545c6eadd56ea)


</details>